lib/oneview-sdk/image-streamer/resource/api300/os_volumes.rb

```diff
@@ -15,8 +15,34 @@ module OneviewSDK
   module ImageStreamer
     module API300
       # OS Volumes resource implementation for Image Streamer
-      class OsVolumes < Resource
+      class OSVolumes < Resource
```

Comment:
Why are you using plural for OSVolumes?
I noticed that usually the names of the resources' classes on the Ruby SDK are in singular.